catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrMetacardClientImpl.java

```diff
@@ -367,9 +371,67 @@ protected SolrQuery postAdapt(
 
     setSortProperty(request, query, filterDelegate);
 
+    filterAttributes(request, query);
+
     return query;
   }
 
+  private void filterAttributes(QueryRequest request, SolrQuery query) {
+    if (skipFilteredAttributes(request)) {
+      return;
+    }
+
+    Set<String> excludedAttributes = (Set<String>) request.getPropertyValue(EXCLUDE_ATTRIBUTES);
+
+    Set<String> excludedFields =
+        resolver
+            .fieldsCache
+            .stream()
+            .filter(Objects::nonNull)
+            .filter(field -> excludedAttributes.stream().anyMatch(field::startsWith))
+            .collect(Collectors.toSet());
+
```

Comment:
At this point we just pulled out all the fields we know about (via the `fieldsCache`) that they want excluded. 

❓ If for some reason they did happen to match any fields (whether valid case or mistake), do we want to add a short circuiting return here? It would essentially just prevent us from sending all our suffixes wildcarded in the field list.